### PR TITLE
Volume - Wrap VolumeControls in a composed modifier

### PR DESCRIPTION
**Pull request by laiyichin**
#### WHAT
- Wrap volume controls in a composed modifier
- Fix RotaryInputAccumulator values when recomposed issue

#### WHY
Persist state for volume control

#### HOW
- Added rotaryVolumeControls as a Modifier
- Used composed and remember in RotaryInputAccumulator

#### Checklist :clipboard:
- [X] Add explicit visibility modifier and explicit return types for public declarations
- [X] Run spotless check
- [X] Run tests
- [X] Update metalava's signature text files

### CODE REVIEW

File: compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryVolumeControls.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+package com.google.android.horologist.compose.rotaryinput
+
+import androidx.compose.foundation.focusable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.wear.compose.foundation.RequestFocusWhenActive
+
+public fun Modifier.rotaryVolumeControls(
```
Comment (yschimke) [inline compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryVolumeControls.kt:26]: Does this need the label volume in it? Or is it and always will be actually generic? Not sure of the right name though.

Comment (laiyichin) [reply compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryVolumeControls.kt]: Updated from rotaryVolumeControls to rotaryControls.

File: audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt [hunk 2]
```diff
@@ -173,32 +173,37 @@ internal fun VolumeScreen(
     increaseIcon: @Composable () -> Unit = { VolumeScreenDefaults.IncreaseIcon() },
     decreaseIcon: @Composable () -> Unit = { VolumeScreenDefaults.DecreaseIcon() },
     showVolumeIndicator: Boolean = true,
-    onVolumeChangeByScroll: ((scrollPixels: Float) -> Unit)
+    onVolumeChangeByScroll: ((scrollPixels: Float) -> Unit)? = null
```
Comment (yschimke) [inline audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt:176]: This seems unfortunate that we are adding back the nullable, didn't we remove?

Comment (laiyichin) [reply audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt]: Yes, this reminds me of the change I actually wanted to do, was to remove this param and move it into the modifier like [PR1018](https://github.com/google/horologist/pull/1018). Thanks!

File: compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryControls.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+package com.google.android.horologist.compose.rotaryinput
+
+import androidx.compose.foundation.focusable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.wear.compose.foundation.RequestFocusWhenActive
+
+public fun Modifier.rotaryControls(
```
Comment (fstanis) [inline compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryControls.kt:26]: Nit: do we need this as a separate function? I wonder if we can just make it an overload of `onRotaryInputAccumulated` or a flag?

**Review by fstanis — APPROVED** [review]

File: compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryControls.kt (hunk 3 above)
Comment (laiyichin) [reply compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/RotaryControls.kt]: Moved it to be with onRotaryInputAccumulated.

**Pull request merged**